README.md

```diff
@@ -25,7 +25,14 @@ steps.forEach(step => {
 
 To solve an equation:
 ```js
-const steps = mathsteps.solveEquation('2x + 2x + x + x');
+const steps = mathsteps.solveEquation('2x + 3x = 35');
+
+steps.forEach(step => {
+    console.log("before change: " + step.oldEquation.print());  // e.g. before change: 2x + 3x = 35
```

Comment:
Ooo true.. I should follow my own rules more haha.

What do you think? Is it a lot more readable on the same line (so we should make an exception) or is better to put them all on the line above? I could go either way :)